Entry 2 -- November 8.md

```diff
@@ -15,12 +17,21 @@ design process.
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+I feel like my API is rudimentary at best. I spent a lot of time this week writing and rewriting the API structure because I couldn't find something I was *too* happy with. Any suggestions in this department would be helpful. Also, ScalaIDE/Eclipse dislikes my program structure and is unable to recognize class names in the same package. I spent some time trying to fix this issue but I think I will have to go to office hours/grutoring to try and fix this one.
+
 **What questions do you have for your critique partners? How can they best help
 you?**
 
+What do you think about the API?
+What kind of API calls do you expect to be able to do through a space management system?
+Do you notice anything blatantly unintuitive in the function calls?
+
 **How much time did you spend on the project this week? If you're working in a
 team, how did you share the labor?**
 
+I spend about 9 hours this week including class studio time.
+About 2 hours spent (wasted?) on trying to fix my project within Eclipse. :-(
+
 ## Post-critique summary
 
 ## Post-critique reflection
```

Comment:
Please remember to fill in these parts of the notebook, to reflect on last week's critique.